src/components/WindowThumbnailSettings.js

```diff
@@ -34,42 +34,60 @@ export class WindowThumbnailSettings extends Component {
    * @return {type}  description
    */
   render() {
-    const { thumbnailNavigationPosition, t } = this.props;
+    const {
+      classes, handleClose, t, thumbnailNavigationPosition,
+    } = this.props;
 
     return (
       <>
-        <Typography>{t('thumbnails')}</Typography>
-        <RadioGroup aria-label={t('position')} name="position" value={thumbnailNavigationPosition} onChange={this.handleChange} row>
+        <ListSubheader role="presentation" tabIndex="-1">
+          <Typography>{t('thumbnails')}</Typography>
+        </ListSubheader>
+
+        <MenuItem className={classes.MenuItem} onClick={() => { this.handleChange('off'); handleClose(); }}>
           <FormControlLabel
```

Comment:
`<FormControlLabel>` seems like an odd choice here, but it gives us the combination of the Icon and Label styled nicely together.  I guess it's not technically any less valid than it was before (given that it never hooked up a proper `for` attribute or anything like that).

If this is unacceptable, I can try to roll something custom for this (or if anybody knows an existing MUI component that would be better, I'm all ears).